app/src/androidTest/java/com/github/h3lp3rs/h3lp/SignInActivityTest.kt

```diff
@@ -1,30 +0,0 @@
-package com.github.h3lp3rs.h3lp
-
-import androidx.test.espresso.Espresso.onView
-import androidx.test.espresso.action.ViewActions
-import androidx.test.espresso.assertion.ViewAssertions.matches
-import androidx.test.espresso.intent.Intents
-import androidx.test.espresso.intent.matcher.IntentMatchers
-import androidx.test.espresso.matcher.ViewMatchers.*
-import androidx.test.ext.junit.rules.ActivityScenarioRule
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import org.junit.Rule
-import org.junit.Test
-import org.junit.runner.RunWith
-
-@RunWith(AndroidJUnit4::class)
-class SignInActivityTest {
-    @get:Rule
-    val testRule = ActivityScenarioRule(
-        SignInActivity::class.java
-    )
-
-    @Test
-    fun loginButtonLaunchesIntent() {
```

Comment:
Just little warning that I used this class to add my own tests (you will see this at merge time anyways)